### PR TITLE
Rid us of manage.py

### PR DESCRIPTION
**Pull request by dwaynebailey**
* Remove from docs as much as possible
* Warning in ./manage.py - sadly suppressed by default in Python
* Various other docs fixes picked up in doing this cleanup

### CODE REVIEW

File: docs/server/commands.rst [hunk 1]
```diff
@@ -1443,7 +1437,7 @@ For the full details on how to configure cron, read your platform documentation
 (for example ``man crontab``). Here is an example that runs the
 :djadmin:`calculate_checks` command daily at 02:00 AM::
 
-    00 02 * * * www-data /var/www/sites/pootle/manage.py calculate_checks
+    00 02 * * * www-data . /home/dwayne/.virtualenvs/mozilla/bin/activate; pootle calculate_checks
```
Comment (unho) [inline docs/server/commands.rst]: mozilla? I would use a different name for the venv.

Comment (unho) [inline docs/server/commands.rst]: Also, is the space after the dot ok?

File: manage.py [hunk 2]
```diff
@@ -9,14 +9,16 @@
...
 import os
 import sys
-
-from django.core.management import execute_from_command_line
+import warnings
 
 from pootle import syspath_override  # noqa
+from pootle.core.log import cmd_log
+
+from django.core.management import execute_from_command_line
```
Comment (unho) [inline manage.py]: If this import doesn't need to be imported after Pootle ones then it is incorrectly placed. If it needs to be here I will put a comment before stating it.

File: manage.py [hunk 3]
```diff
@@ -9,14 +9,16 @@
...
 
 
 if __name__ == "__main__":
-    from pootle.core.log import cmd_log
+    warnings.warn("Deprecated use 'pootle' command instead", DeprecationWarning)
```
Comment (unho) [inline manage.py]: `"Deprecated. Use 'pootle' command instead."`

File: pootle/runner.py [hunk 4]
```diff
@@ -14,11 +14,13 @@
 import sys
 from argparse import SUPPRESS, ArgumentParser
 
+from pootle.core.cache import PERSISTENT_STORES
+from pootle.core.log import cmd_log
```
Comment (unho) [inline pootle/runner.py]: If these imports need to be before the Django ones I will put a comment stating it clearly.

File: pootle/tools/docs_check_commands.sh [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
-    $BASE_DIR/manage.py --help | \
+    pootle | \
     sed -E -n "/^\[$app\]$/,/^\[/ p" | \
     sed "/^\[/d" | \
     sed "s/^[ ]*//g"
 done)
 
 django_commands=$(
-    $BASE_DIR/manage.py --help | \
+    pootle | \
```
Comment (unho) [inline pootle/tools/docs_check_commands.sh]: Is `--help` not necessary here and above?

File: docs/server/commands.rst [hunk 6]
```diff
@@ -579,6 +579,17 @@ Gets, sets, lists, appends and clears pootle configuration settings.
...
 
+.. django-admin:: schema
+
+schema
+^^^^^^
+
+.. versionadded:: 2.8
+
+Dumps a JSON representation for the Pootle database schema, currently only
+MySQL, for debugging and comparision to a reference database schema.
```
Comment (unho) [inline docs/server/commands.rst]: Typo: comparison.

**Comment by unho** [comment]
The major concern is the order of the imports. Other than that there only the raised nits. Once those fixed lgtm.

File: pootle/tools/docs_check_commands.sh (hunk 5 above)
Comment (dwaynebailey) [inline pootle/tools/docs_check_commands.sh]: Specifically not in fact, --help give help, nothing gives list of possible commands.  Odd but true.

File: docs/server/commands.rst (hunk 1 above)
Comment (dwaynebailey) [inline docs/server/commands.rst]: `. blah` == `source blah` and yes it is correct, it is != `./`

File: manage.py (hunk 2 above)
Comment (dwaynebailey) [inline manage.py]: Sorted

**Pull request merged**